pyscript.core/tests/integration/test_async.py

```diff
@@ -121,10 +123,52 @@ async def a_func():
         inner_text = self.page.inner_text("html")
         assert "A0\nA1\nB0\nB1" in filter_inner_text(inner_text)
 
+    @pytest.mark.skip("ERROR_SCRIPT: works with <py-script> not with <script>. "
+                      "it seems that the 2nd task is never created, "
+                      "but note that it works if you use <py-script> instead")
+    def test_multiple_async_multiple_display_targeted_script_py(self):
+        self.pyscript_run(
+            """
+                <script type="py"'>
+                    from pyscript import display
+                    import js
+                    import asyncio
+
+                    async def a_func():
+                        for i in range(2):
+                            #display(f'A{i}', target='pyA', append=True)
+                            js.console.log("A", i)
+                            await asyncio.sleep(0.1)
+                    asyncio.ensure_future(a_func())
+
+                </script>
+                <div id='pyA"></div>
```

Comment:
Mismatched quotes here are causing a parsing error; so the browser never sees/opens the second script tag. Fixing this and restoring the `display` calls lets the test pass.

Reply:
yes, I just re-discovered it and came here to write... you were faster!
Glad that it was such a silly mistake and nothing more serious! :)

Reply:
fixed by 2f8e81e (will push it soon)